niworkflows/interfaces/cifti.py

```diff
@@ -15,47 +16,62 @@
     BaseInterfaceInputSpec, TraitedSpec, File, traits,
     SimpleInterface, Directory
 )
-from templateflow.api import get as get_template
+import templateflow.api as tf
 
-# CITFI structures with corresponding FS labels
-CIFTI_STRUCT_WITH_LABELS = {
+CIFTI_SURFACES = ("fsaverage5", "fsaverage6", "fsLR")
+CIFTI_VOLUMES = ("MNI152NLin2009cAsym", "MNI152NLin6Asym")
+CIFTI_STRUCT_WITH_LABELS = {  # CITFI structures with corresponding labels
     # SURFACES
     'CIFTI_STRUCTURE_CORTEX_LEFT': None,
     'CIFTI_STRUCTURE_CORTEX_RIGHT': None,
 
     # SUBCORTICAL
-    'CIFTI_STRUCTURE_ACCUMBENS_LEFT': [26],
-    'CIFTI_STRUCTURE_ACCUMBENS_RIGHT': [58],
-    'CIFTI_STRUCTURE_AMYGDALA_LEFT': [18],
-    'CIFTI_STRUCTURE_AMYGDALA_RIGHT': [54],
-    'CIFTI_STRUCTURE_BRAIN_STEM': [16],
-    'CIFTI_STRUCTURE_CAUDATE_LEFT': [11],
-    'CIFTI_STRUCTURE_CAUDATE_RIGHT': [50],
-    'CIFTI_STRUCTURE_CEREBELLUM_LEFT': [6],
-    'CIFTI_STRUCTURE_CEREBELLUM_RIGHT': [45],
-    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_LEFT': [28],
-    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_RIGHT': [60],
-    'CIFTI_STRUCTURE_HIPPOCAMPUS_LEFT': [17],
-    'CIFTI_STRUCTURE_HIPPOCAMPUS_RIGHT': [53],
-    'CIFTI_STRUCTURE_PALLIDUM_LEFT': [13],
-    'CIFTI_STRUCTURE_PALLIDUM_RIGHT': [52],
-    'CIFTI_STRUCTURE_PUTAMEN_LEFT': [12],
-    'CIFTI_STRUCTURE_PUTAMEN_RIGHT': [51],
-    'CIFTI_STRUCTURE_THALAMUS_LEFT': [10],
-    'CIFTI_STRUCTURE_THALAMUS_RIGHT': [49],
+    'CIFTI_STRUCTURE_ACCUMBENS_LEFT': (26,),
+    'CIFTI_STRUCTURE_ACCUMBENS_RIGHT': (58,),
+    'CIFTI_STRUCTURE_AMYGDALA_LEFT': (18,),
+    'CIFTI_STRUCTURE_AMYGDALA_RIGHT': (54,),
+    'CIFTI_STRUCTURE_BRAIN_STEM': (16,),
+    'CIFTI_STRUCTURE_CAUDATE_LEFT': (11,),
+    'CIFTI_STRUCTURE_CAUDATE_RIGHT': (50,),
+    'CIFTI_STRUCTURE_CEREBELLUM_LEFT': (
+        6,  # DKT31
+        8,  # HCP MNI152
+    ),
+    'CIFTI_STRUCTURE_CEREBELLUM_RIGHT': (
+        45,  # DKT31
+        47,  # HCP MNI152
+    ),
+    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_LEFT': (28,),
+    'CIFTI_STRUCTURE_DIENCEPHALON_VENTRAL_RIGHT': (60,),
+    'CIFTI_STRUCTURE_HIPPOCAMPUS_LEFT': (17,),
+    'CIFTI_STRUCTURE_HIPPOCAMPUS_RIGHT': (53,),
+    'CIFTI_STRUCTURE_PALLIDUM_LEFT': (13,),
+    'CIFTI_STRUCTURE_PALLIDUM_RIGHT': (52,),
+    'CIFTI_STRUCTURE_PUTAMEN_LEFT': (12,),
+    'CIFTI_STRUCTURE_PUTAMEN_RIGHT': (51,),
+    'CIFTI_STRUCTURE_THALAMUS_LEFT': (10,),
+    'CIFTI_STRUCTURE_THALAMUS_RIGHT': (49,),
+}
+CIFTI_VARIANTS = {
+    'HCP grayordinates': ('fsLR', 'MNI152NLin6Asym'),
+    'space1': ('fsaverage5', 'MNI152NLin2009cAsym'),
```

Comment:
Can we maybe merge space1 and space2 with:
```suggestion
    'fMRIPrep grayordinates': ('fsaverage', 'MNI152NLin2009cAsym'),
```

The idea would be that density could be encoded in the filenames, and this would be more consistent.